Makefile

```diff
@@ -36,7 +36,7 @@ install: godep gui ## Install chainlink
 	go install $(GOFLAGS)
 
 gui: yarndep ## Install GUI
-	yarn build
+	CHAINLINK_VERSION=$(shell chainlink --version) yarn build
```

Comment:
I don't think this will work for a first-time user running `make install`. Even on my machine with a previous install existing, I receive the following for `make gui`:
```
yarn install
yarn install v1.6.0
[1/4] Resolving packages...
success Already up-to-date.
Done in 1.14s.
CHAINLINK_VERSION=chainlink version 0.2.0 yarn build
/bin/sh: 1: version: not found
Makefile:39: recipe for target 'gui' failed
make: *** [gui] Error 127
```